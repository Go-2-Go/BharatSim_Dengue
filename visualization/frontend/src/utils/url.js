const DATASOURCE_URL = '/api/dataSources';

export const url = {
  getDataUrl: (dataSource) => `${DATASOURCE_URL}/${dataSource}/data`,
  getHeaderUrl: (dataSource) => `${DATASOURCE_URL}/${dataSource}/headers`,
  DATA_SOURCES: `${DATASOURCE_URL}`,
};
