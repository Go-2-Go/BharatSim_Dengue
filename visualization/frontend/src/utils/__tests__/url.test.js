import { url } from '../url';

describe('URL', () => {
  it('should provide a url', () => {
    expect(url).toMatchSnapshot();
  });
});
