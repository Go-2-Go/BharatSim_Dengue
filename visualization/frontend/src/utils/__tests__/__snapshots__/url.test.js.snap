// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`URL should provide a url 1`] = `
Object {
  "DATA_SOURCES": "/api/dataSources",
  "getDataUrl": [Function],
  "getHeaderUrl": [Function],
}
`;
